include/deal.II/sundials/arkode.h

```diff
@@ -975,6 +975,27 @@ namespace SUNDIALS
      */
     LinearSolveFunction<VectorType> solve_linearized_system;
 
+
+    /**
+     * Set the given @p solver for the solution of the linearized system.
+     *
+     * @note Alternative to setting solve_linearized_system directly.
+     */
+    template <typename SolverType>
+    void
+    set_linear_solver(SolverType &solver);
```

Comment:
Not sure if this function is really needed. I guess one always has access to the solver control as well? With this function SUNDIALS cannot set its calculated tolerance on the solver.